src/Student8/Ex1/Menu.scss

```diff
@@ -0,0 +1,15 @@
+h1 {
+    background-color: cyan;
+}
+
+.Food {
```

Comment:
I think we can remove this because it's already defined in Food.scss right?  I think we can also remove the classes for Drink and Dessert below, right?